pandas/core/window/rolling.py

```diff
@@ -646,8 +649,12 @@ def _numba_apply(
             out = obj._constructor(result, index=index, columns=columns)
             return self._resolve_output(out, obj)
 
-    def aggregate(self, func, *args, **kwargs):
+    def aggregate(self, func=None, *args, **kwargs):
+        relabeling, func, columns, order = reconstruct_func(func, **kwargs)
         result = ResamplerWindowApply(self, func, args=args, kwargs=kwargs).agg()
+        if (result is not None) and isinstance(result, ABCDataFrame) and relabeling:
```

Comment:
```suggestion
        if result is not None and isinstance(result, ABCDataFrame) and relabeling:
```

Reply:
@mroeschke  thanks for the suggestion. This particular line seems to fail mypy so I had to play around with it a little bit despite no success. Any idea why it throws this error?
```
mypy.....................................................................Failed
- hook id: mypy
- duration: 83.06s
- exit code: 1

pandas/core/window/rolling.py:657: error: Item "None" of "DataFrame | Series | None" has no attribute "columns"  [union-attr]
Found 1 error in 1 file (checked 1446 source files)
```

Reply:
The provided suggestion might narrow the type enough to fix the mypy error, or else you can add a type ignore in this case